packages/oasgraph-cli/README.md

```diff
@@ -19,29 +19,49 @@ npm i -g oasgraph-cli
 
 
 ## Usage
-The basic usage of the CLI takes the specified OAS, create a GraphQL interface for it, and start a server to host the GraphQL interface.
 
-You can specify a local file containing the OAS specification or a remote url such as `http://127.0.0.1:3000/openapi.json`. Additionally, you can specify an optional port number so you can have multiple GraphQL servers for testing in the same machine.
+```
+Usage: oasgraph <OAS JSON file path or remote url> [options]
+
+Options:
+  -V, --version             output the version number
+  -p, --port <port>         select the port where the server will start
+  -s, --strict              throw an error if OASGraph cannot run without compensating for errors or missing data in the OAS
+  -a, --addSubOperations    nest operations based on path hierarchy
+  -f, --fillEmptyResponses  run OASGraph on operations with no response schema, such as those with HTTP 204 code, rather than ignore them
```

Comment:
I propose: `create placeholder object type for operations with HTTP status code 204 (no response) rather than ignore them`.